src/printer/relax_script_printer.cc

```diff
@@ -354,22 +354,30 @@ Doc RelaxScriptPrinter::VisitNode_(const relax::ExternFuncNode* op) {
   return Doc::StrLiteral(op->global_symbol);
 }
 
+Doc RelaxScriptPrinter::PrintPrimExpr(const PrimExpr& expr) {
+  Doc body = VisitExpr(expr);
+  if (print_symbolic_shape_as_str_ && !expr->IsInstance<IntImmNode>()) {
```

Comment:
Hmmm should this be `expr` or `body`? 
```suggestion
  if (print_symbolic_shape_as_str_ && !body->IsInstance<IntImmNode>()) {
```

Reply:
`body` is a Doc, not PrimExpr here :). Note that it's a ExprFunctor -> Doc, not a ExprMutator